portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,7 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
We typically only import the packages that are needed, instead of importing *. This avoid unnecessary build dependencies. Here is a Google Java style guide, it is too much, only need to look up when you are unsure about something https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports.

import java.util.ArrayList;
import java.util.Arrays;

Reply:
I will do that in the future. Done!